e2e/tests/integration/simulation-export.ts

```diff
@@ -109,7 +109,7 @@ describe('Simulation export', () => {
                     const nonEeaDeskPaxCount = sumColumn(csvData, nonEeaDeskPaxCsvIndex)
                     const eGatePaxCount = sumColumn(csvData, eGatePaxCsvIndex)
 
-                    expect(eeaDeskPxCount).to.equal(4, "expected 2 passengers in EEA with weighting of 2")
+                    expect(eeaDeskPxCount).to.equal(4, "expected 4 passengers in EEA with weighting of 2")
```

Comment:
Do you know why you need to change the expected test result here? We may have introduced a bug

Reply:
It is just description change , if you see expected is 4 where as description was saying 2  so just corrected it